static/app/views/replays/detail/console/messageFormatter.tsx

```diff
@@ -44,10 +44,10 @@ function UnmemoizedMessageFormatter({frame, expandPaths, onExpand}: Props) {
     );
   }
 
-  const args = isConsoleFrame(frame) ? frame.data.arguments : undefined;
```

Comment:
This is already narrowed to `ConsoleFrame` by the above `if (!isConsoleFrame(frame)` condition.

That condition also forces `frame.data = {}` so we fix the bug!